### ISSUE TITLE
End of City of London Event

### ISSUE THREAD

**Issue by omi20** [simsulla/nwomod#1188]
Is there a way to prevent it from trigerring or changing it in the future

**Comment by simsulla** [comment]
Yes. the decision is triggered if you are a greater power and has special_drawing_rights (which is a other decision) and the UK does not have 100000£ or is in bankruptcy.

**Comment by omi20** [comment]
so can it  be stopped at all
